common/searchattribute/mapper.go

```diff
@@ -39,11 +40,58 @@ type (
 		GetAlias(fieldName string, namespace string) (string, error)
 		GetFieldName(alias string, namespace string) (string, error)
 	}
+
+	identityMapper struct{}
```

Comment:
`passThroughMapper`, `noopMapper`? `identity` is probably the best but it is so overloaded with another meanings.

Reply:
Renamed to `noopMapper`.